beacon_node/network/src/router/processor.rs

```diff
@@ -289,8 +301,15 @@ impl<T: BeaconChainTypes> Processor<T> {
         request_id: PeerRequestId,
         mut req: BlocksByRangeRequest,
     ) {
+        let chain = self.chain.clone();
+        let mut network = self.network.clone();
+        let log = self.log.clone();
+
+        // Shift the db reads to a blocking thread.
+        self.executor.spawn_blocking(move || {
+
         debug!(
```

Comment:
I think this whole block should be indented. I guess rustfmt is struggling with the logging macros